src/test/java/com/beust/jcommander/JCommanderTest.java

```diff
@@ -1447,4 +1447,29 @@ class Args {
     public static void main(String[] args) {
         new JCommanderTest().trimTest();
     }
+
+    @Test
+    public void ignoreDefaultValueForRequiredParameter() {
+        class Parameters {
+            @Parameter(names = "-v",
+                    required = true,
+                    validateValueWith = StrictlyPositiveInteger.class)
+            private int value;
+        }
+
+        String[] argv = { "-v", "1" };
+        Parameters args = new Parameters();
+        JCommander.newBuilder().addObject(args).build().parse(argv);
```

Comment:
Looks like a duplicate line. The next line does exactly the same.

Reply:
Sorry, I've copied / pasted this block:
https://github.com/cbeust/jcommander/blob/748ddba20a28956332b413eaa99d7e8b5422104d/src/test/java/com/beust/jcommander/JCommanderTest.java#L1297-L1311

Reply:
Can you please update the PR?

Reply:
See 632dfe0
I've fixed only my lines.
And not https://github.com/cbeust/jcommander/blob/748ddba20a28956332b413eaa99d7e8b5422104d/src/test/java/com/beust/jcommander/JCommanderTest.java#L1308-L1310